gen/definitions/properties.yaml

```diff
@@ -220,3 +220,28 @@ l3extRsOutToFBRGroup:
       parent_dependency_dn: "aci_vrf.test.id"
       parent_dn: "aci_vrf_fallback_route_group.test.id"
       target_dn: "uni/tn-test_tenant/ctx-test_vrf/fbrg-fallback_route_group"
+
+l3extProvLbl:
+  resource_required:
+    - "name"
+  datasource_required:
+    - "name"
```

Comment:
Do we need those? name is a required field for those so not sure if we need this in properties.yaml.